pkg/hookbot/hookbot.go

```diff
@@ -115,12 +115,19 @@ func (h *Hookbot) Shutdown() {
 	h.wg.Wait()
 }
 
-// Returns "true" if fullTopic ends with `?recursive` and returns topic name
-// without `?recursive` suffix.
+// Returns "true" if fullTopic ends with a "/".
+// Also supports topics ending with ?recursive to support legacy infrastructure.
 func recursive(fullTopic string) (topic string, isRecursive bool) {
+	// Legacy: topics ending in ?recursive are recursive.
```

Comment:
I considered this, but that would require updating dinger at the same time and I can't guarantee there isn't something using it. I first want to check that at least hanoverd responding to github and slack notification continues to work with the first set of changes before removing this, if that's OK.